psutil/_psutil_linux.c

```diff
@@ -48,41 +48,44 @@ static PyMethodDef mod_methods[] = {
 };
 
 
-static struct PyModuleDef moduledef = {
-    PyModuleDef_HEAD_INIT,
-    "_psutil_linux",
-    NULL,
-    -1,
-    mod_methods,
-    NULL,
-    NULL,
-    NULL,
-    NULL
-};
-
-
-PyObject *
-PyInit__psutil_linux(void) {
-    PyObject *mod = PyModule_Create(&moduledef);
-    if (mod == NULL)
-        return NULL;
-
-#ifdef Py_GIL_DISABLED
-    if (PyUnstable_Module_SetGIL(mod, Py_MOD_GIL_NOT_USED))
-        return NULL;
-#endif
```

Comment:
This was added by @colesbury in https://github.com/giampaolo/psutil/commit/73af40731. Is it OK to delete it?

Reply:
Yeah, it's fine to remove. The `{Py_mod_gil, Py_MOD_GIL_NOT_USED},` in the slots is the equivalent for multi-phase init.

Reply:
Thank you. ;)